src/averageAges.js

```diff
@@ -37,7 +45,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenAges = people.filter(human => withChildren
+    ? (withChildren === people.some(person => person.mother === human.name))
```

Comment:
why do you need this here? 
`withChildren === people.some(person => person.mother === human.name)`